chat-agent/README.md

```
@@ -144,12 +146,14 @@ Health check endpoint.
 Send a message to the AI chat agent.
 
 **Headers:**
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)